cli/progress.c

```diff
@@ -83,7 +83,7 @@ static void print_bar(int cur, int total)
 	int pos;
 
 	bar_width = get_columns() - 33;
-	pos = bar_width * cur / total;
+	pos = (long long)bar_width * cur / total;
```

Comment:
Why not make bar width a (long long) instead of the ugly cast?

Not sure swithctec images are ever greater than 40MB, but I guess it would be good to fix it here.